documentation/api-gremlin.md

```diff
@@ -452,8 +468,8 @@ G.V().Has('Type', Without('ovsbridge', 'ovsport'))
   expression.
 
 {% highlight shell %}
-G.V().Has('Name', Regex('tap.*'))
+G.V().Has('Name', Regex("tap.*"))
```

Comment:
mm, single quote is supported maybe your issue comes from the mix of single quote/double quote in the command line

Reply:
@safchain you are right, I tried it, and simple quotes and they just work as well. Removed from the PR.